scona/visualisations.py

```diff
@@ -0,0 +1,397 @@
+import warnings
+
+import numpy as np
+import networkx as nx
+import pandas as pd
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+from scona.helpers import save_fig
+
+
+def plot_rich_club(brain_bundle, real_network, figure_name=None, color=None,
```

Comment:
Hi @wingedRuslan! I didn't have a huge amount of time over the weekend so I'm just doing a quick look through. One thing I've noticed is that there are some options that should be passed to this function as arguments to control the plotting. Eg for the legend and for the confidence intervals.

Can we make sure that its easy to control the legend text in the same way across functions? So however you'd adjust the legend text for the plot_network_measures function that should be the same for plot_rich_club.

See also #139 #138 

Thank you!

Reply:
#138 is resolved, although it has spun into a new complaint of my own about terminology